### DISCUSSION TITLE
Frontend Initiatives Meeting - June 8, 2021

### DISCUSSION THREAD

**Discussion by varl** [dhis2/notes#291] [Meetings] [answered]
When: 2021-06-08 @ 15:00 CET
Where: Remote

# Agenda

- General updates
  - Proposal for how [application versions](https://github.com/dhis2/notes/discussions/293) will be handled in the CI/CD world in progress
  - DataTable has been released in UI along with new features: https://github.com/dhis2/ui/blob/master/CHANGELOG.md
- Lighthouse testing on CI (see [slack](https://dhis2.slack.com/archives/C0BP0RABF/p1623050297078300)).
  - If everyone likes this we could add it to Jira.
- SameSite cookie changes in Chrome
- Work groups updates
  - Cypress
  - PWA
  - Developer Advocacy
  - Client-side caching
    - Outcome of the proposals: https://github.com/dhis2/notes/blob/master/minutes/2021/06/03-client-side-caching.md
  - PR Previews
    - New workflows available, can be [installed with `d2-style`](https://dhis2.slack.com/archives/C0BP0RABF/p1622810229070700) 
    - To do: updating the default CORS whitelist in databases and environments.
      - Done for debug.dhis2.org/dev
    - Example workflow: https://github.com/dhis2/workflows/blob/master/ci/dhis2-preview-pr.yml
  - Visual regression testing
  - i18n
  - Browserlist
- Misc?

**Comment by ghost** [comment]
- Lighthouse testing on CI (see [slack](https://dhis2.slack.com/archives/C0BP0RABF/p1623050297078300)). If everyone likes this we could add it to Jira.

**Comment by varl** [comment, marked as answer]
Meeting minutes: https://github.com/dhis2/notes/pull/294
